From ef8a7c486de273ced4bd50937949dd8791db846c Mon Sep 17 00:00:00 2001
From: Simon Shields <simon@lineageos.org>
Date: Sat, 20 Jan 2018 19:52:51 +1100
Subject: [PATCH 185/249] wiki: use github as image host for CI

this saves us from hosting ~70MB of images per active CI output.

Use: JEKYLL_ENV=<commit sha1> jekyll build

Change-Id: Id587385e897c3aab0a696542c311e2b9e0d2ec7b
---
 _includes/deviceinfo.html | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/_includes/deviceinfo.html b/_includes/deviceinfo.html
index b618b0d..8ec35d4 100644
--- a/_includes/deviceinfo.html
+++ b/_includes/deviceinfo.html
@@ -26,7 +26,11 @@ ul, ol {
         <td colspan="2" class="header"><strong>{{ device.vendor }} {{ device.name }} ({{ device.codename }})</strong></td>
     </tr>
     <tr>
+        {% if jekyll.environment == "development" or jekyll.environment == "production" %}
         <td colspan="2"><img style="width:100%" src="{{ "images/devices/" | append: device.image | relative_url }}" /></td>
+        {% else %}
+        <td colspan="2"><img style="width:100%" src="{{ "https://raw.githubusercontent.com/LineageOS/lineage_wiki/" | append: jekyll.environment | append: "/images/devices/" | append: device.image }}"/></td>
+        {% endif %}
     </tr>
     <tr>
         <td class="header">Released</td>
-- 
2.17.1

